tasks/grunt-postmark-templates.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * grunt-postmark-templates
+ * https://github.com/wildbit/grunt-postmark.git
+ */
+
+module.exports = function(grunt) {
+
+  'use strict';
+
+  grunt.registerMultiTask('postmark-templates', 'Push templates', function() {
+
+    var done = this.async();
+    var options = this.options();
+    var _data = this.data;
+
+    // Check for server token
+    if (!options.serverToken && !_data.serverToken) {
+      grunt.fail.warn('Missing required option "serverToken" \n');
+    }
+
+    // Check for required attributes
+    // TODO: require one of fooBody or fooFile
+    ['name', 'subject'].forEach(function(name){
+      requireProperty(name, options, _data);
+    });
+
+    // Postmark lib
+    var postmark = require('postmark');
+    var client = new postmark.Client(options.serverToken || _data.serverToken);
+    var htmlBody = _data.htmlBody || options.htmlBody;
+    var textBody = _data.textBody || options.textBody;
+
+    if (_data.htmlFile) {
+      htmlBody = grunt.file.read(_data.htmlFile);
+    }
+    if (_data.textFile) {
+      textBody = grunt.file.read(_data.textFile);
+    }
+    client.createTemplate({
+        name: _data.name || options.name,
+        textBody: textBody,
+        htmlBody: htmlBody,
+        subject: _data.subject || options.subject,
+    }, function(err, response) {
+      handleResponse(err, response, done);
```

Comment:
> I'd like to get editTempate (update) working as well, but haven't figured out where the templateIDs will come from. It might make sense to move this to a separate grunt plugin.

Can you see the `TemplateId` being returned from this [response object](http://developer.postmarkapp.com/developer-api-templates.html#create-template)? From here, could you maintain a reference to the email template in the file system, the server(s) it lives on, and template ID of server in an external JSON file? I know it's a bit tough to manage on the plugin end, but it would be super nice for users to not even think about this.